src/Sql/Sql/help/Update-AzSqlSyncMember.md

```diff
@@ -53,6 +53,8 @@ This command resets the administrator password for the member database.
 Updates an Azure SQL Database Sync Member. (autogenerated)
 
 <!-- Aladdin Generated Example -->
+
+
```

Comment:
```suggestion
```
Please check all your changes to avoid these blank changes.

Reply:
I actually haven't edited these files by hand, changes were introduced by platyPS 
(https://github.com/Azure/azure-powershell/pull/17422/commits/12d8577018955545e1673ce9a11e8560f92b67e7)

So yeah I can revert but I guess it will reoccur next time someone runs platyps